Algorithm-Convert-HTML-Entities.md

```diff
@@ -101,7 +101,7 @@ function convertHTML(str) {
 convertHTML("Dolce & Gabbana");
 ```
 
-:rocket: [REPL It!](https://repl.it/CLnR/0)
+:rocket: [Run Code](https://repl.it/CLnR/0)
 
 ## Code Explanation (Advanced Solution):
```

Comment:
@Rafase282 can you check why this file in particular has this kind of weird highlighting? maybe we have missed it earlier